AEPLifecycle/Tests/FunctionalTests/LifecycleFunctionalTests.swift

```diff
@@ -337,6 +337,47 @@ class LifecycleFunctionalTests: XCTestCase {
         XCTAssertEqual(startEvent2.timestamp.timeIntervalSince1970, storedUpgradeDate?.timeIntervalSince1970)
         XCTAssertNotEqual(storedInstallDate, storedUpgradeDate)
     }
+    
+    /// Tests upgrade event when app version number changes
+    func testUpgradeWhenVersionNumberChanged() {
```

Comment:
I find this test quite confusing as it does not clearly state the expectation. can we update to something like this
```
// tests upgrade event is not triggered when app version number changes (it should only trigger on build number change)
testUpgrade_WhenVersionNumberChanged_doesNotTriggerNewUpgrade

Reply:
The newly added tests follow the same naming pattern as existing tests. I will work on updating them separately.